lab_3_ann_retriever/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "target_score": 0
+    "target_score": 4
```

Comment:
good start, but you should hurry up! Make CI green, fix all comments and continue your work. You can reach 8!

Reply:
in progress (thank you)